pkgs/development/libraries/science/math/magma/generic.nix

```diff
@@ -148,13 +157,6 @@ stdenv.mkDerivation {
     "-DMAGMA_ENABLE_HIP=ON"
   ];
 
-  # NOTE: The stdenv's CXX is used when compiling the CMake test to determine the version of
-  #   CUDA available. This isn't necessarily the same as cudatoolkit.cc, so we must set
-  #   CUDAHOSTCXX.
-  preConfigure = strings.optionalString cudaSupport ''
-    export CUDAHOSTCXX=${cudatoolkit.cc}/bin/c++
-  '';
-
```

Comment:
Removed because it's now set in `cuda_nvcc`: https://github.com/NixOS/nixpkgs/blob/a59944f1c11a5e1a6bd33cad379dae257d4bf40b/pkgs/development/compilers/cudatoolkit/redist/overrides.nix#L52 and https://github.com/NixOS/nixpkgs/blob/a59944f1c11a5e1a6bd33cad379dae257d4bf40b/pkgs/development/compilers/cudatoolkit/common.nix#L163